.github/workflows/test.yml

```diff
@@ -19,9 +35,15 @@ jobs:
         run: yarn lint
       - name: Run tests
         run: yarn test
-      - name: Run tests
-        run: yarn test:e2e
+      - name: Run E2E tests
+        run: yarn test:e2e:docker
         env:
           PROVIDERS_URLS: ${{ secrets.PROVIDERS_URLS }}
+          CL_API_URLS: "https://e2e-test.lido.fi,"
```

Comment:
why ?

Reply:
maybe instead we should set VALIDATOR_REGISTRY_ENABLE=false

Reply:
This code will fail due to spread operators for `CL_API_URLS` and `PROVIDERS_URLS`:
https://github.com/lidofinance/lido-keys-api/blob/9d800c9eac0270c42a00cbea09fbb111c4e5026c/src/common/config/config.service.ts#L9-L13

Perhaps, this PR is not a good place to change this. We can create a task for that.

Reply:
it is strange, we have VALIDATOR_REGISTRY_ENABLE=false psecially not to provide CL_API_URLS
i dont think it will fail

Reply:
<img width="894" alt="image" src="https://github.com/lidofinance/lido-keys-api/assets/6780288/1bbf73aa-55f9-4af5-abc2-cf79485a5a5e">
https://github.com/lidofinance/lido-keys-api/actions/runs/6796032612/job/18475289284

without `CL_API_URLS`

Reply:
did you add VALIDATOR_REGISTRY_ENABLE=false ?

Reply:
it is important to add VALIDATOR_REGISTRY_ENABLE=false for such setup

Reply:
i see, for validators tests it is impossible to set VALIDATOR_REGISTRY_ENABLE = false

Reply:
sorry, before i mocked config , so it worked, now you are right